guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+    - file: cashu-id5
+      modalImage: cashu-id5@2x
+      alt: Example of supported features display in a wallet interface
+      caption: Supported features are pulled by indexing a mints information and reading what nuts the mint has implemented.
+
+---
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/best-practicies.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/best-practicies@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/best-practicies-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/best-practicies-mobile@2x.jpg"
+   alt-text = "ecash best practices illustration"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+<!--
+
+
+This is a summary page of best design practices for both bitcoin-backed ecash protocols: Cashu and Fedimint.
+
+The design source for screen mock-ups on this page can be found here:
+ADD Figma URL
+
+
+-->
+
+# Ecash design best practices
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+## General Ecash Best Practices
+When designing bitcoin-backed ecash applications it's important to prioritize clear and intuitive interfaces that allow users to easily manage their tokens, whether they are minting, sending, or redeeming them.
+
+### Multiple Mint or Federation Display
+{% include image-gallery.html pages = page.images_mint-list %}
+
+When displaying multiple mints or federations in a bitcoin-backed ecash application, there are several best practices to consider:
+
+1. **Clear Hierarchy and Organization**: Ensure that mints or federations are organized in a way that is easy for users to navigate. Group mints by relevance, trust level, or frequency of use, and consider using sorting or filtering options to help users find the right mint or federation quickly.
+
+2. **Prominent Metadata Display**: Use metadata fields like mint name, icon, URL, and description, to give users insight into each mint or federations identity. Display this information prominently, so users can easily verify the authenticity and relevance of each mint or federation.
+
+3. **Quick Mint Access**: Provide easy access to mint or federation details directly from the mint or federation list. This allows users to interact with mints or federations more fluidly without needing to navigate through multiple screens.
+
+#### Pending Tokens
+{% include image-gallery.html pages = page.images_pending %}
+
+It's useful to display generated tokens in the transaction history along with their status—whether pending or redeemed. This helps users quickly assess their transaction status. Providing the ability to expand on a token to view details and take actions like resending or redeeming is also important. This feature becomes particularly valuable if a token was lost or never claimed.
```

Comment:
A generated token is simply a token created by the user. Wallets display tokens as 'pending' or 'redeemed' depending on whether they've been redeemed, usually they use an hourglass or a cock icon for 'pending' tokens. Sometimes, a user might issue a token as a gift or payment, but the recipient may not claim it, or they may forget about it. In such cases, if the sender wants to issue a new token from a different mint, it would be useful for the original sender to have the ability to redeem the token they issued and recover the funds.